nvmeadm/tests/discovery_test.rs

```diff
@@ -92,7 +91,7 @@ pub struct NvmfTarget {
 }
 
 impl NvmfTarget {
-    pub fn new(config_file: &str, nvmf_port: &str) -> Result<Self, Error> {
+    pub fn new(config_file: &str, nvmf_port: &str) -> Result<Self, ()> {
```

Comment:
guess you could just return Self as it looks like this fn never fails (just panics)

Reply:
Done.